content/admin/policy_mgmt/VulnerabilityManagement.md

```diff
@@ -0,0 +1,67 @@
+<!--
+title: "Vulnerability Management"
+description: "Overview of vulnerability management"
+tags: "Admin vulnerability policy management"
+-->
+
+> **Note:** This Vulnerability Management page consolidates the information regarding the Vulnerability Behavior section, which was previously included as part of the Applications Settings page. Additionally, the Remediation Policy page no longer exists and has been replaced by this page.
+
+Control how remediation happens. Define if vulnerabilities of a certain severity require approval to have their statuses change, or if you’d rather Contrast close them automatically for you. Go to the **User menu > Policy Management > Vulnerability Management** to get started.
+
+## Vulnerability Behavior 
+
+Check the box to **Require administrative approval when closing vulnerabilities** in your organization. In the fields below, choose the statuses and severities of vulnerabilities that should automatically go into a Pending state when a user moves to close them. When a user requests to close any qualifying vulnerabilities, Contrast will [notify](admin-orgsettings.html#org-notify) you that your review is needed.
+
+<a href="assets/images/Default-vuln-behavior.png" rel="lightbox" title="Define Vulnerability Behavior"><img class="thumbnail" src="assets/images/Default-vuln-behavior.png"/></a>
+
+> **Note:** To qualify for administrative approval, both a status and severity that you select in this configuration must apply to the vulnerability being closed. 
+
+Each vulnerability status will remain pending until you submit your review of the closure. If you deny the closure of a vulnerability, you must provide a reason for denial. Once confirmed, your feedback appears in the vulnerability's **Discussion** tab. If you disable the feature, any pending closures are automatically approved. 
+
+> **Note:** While in a Pending state, the vulnerability's previous status still applies for the purpose of organizational reports and statistics. 
+
+See [Manage Vulnerabilities](user-vulns.html#manage-vuln) for more information about Pending states and workflows.
+
+## Vulnerability Policy
+
+Create policies for vulnerabilities in your organization, and Contrast will clean up your view of security risks. As Contrast assesses your applications, it can either flag vulnerabilities that are in violation of the policy or automatically mark them Remediated - Auto-Verified. 
+
+### How It Works 
+
+
+Administrators can define requirements for vulnerability remediation based on any vulnerability rule, severity, application(s) and route which should comply. Contrast then marks any vulnerabilities that are in violation or Remediated - Auto-Verified based on this policy with a notification in the interface. Administrators are also notified of violations in the product and by email.
```

Comment:
Consider bolding  vulnerability rule, severity, application(s) and route. It's done in the Add policies section below where they are referred to as entry fields.